glrm/glrm.py

```diff
@@ -60,17 +64,18 @@ def _initialize_probs(self):
         self.objY = 0
         Zx = self.Xv @ self.Yp
         Zy = self.Xp @ self.Yv
-        for columns,loss_fxn in self.loss_list:
-            for col in columns:
+        columns = self.A.shape[1]
+        for loss_fxn in self.loss_list:
+            for col in range(columns):
```

Comment:
This is my bad for not documenting this but I'm pretty sure that this was intentional and that I wanted `loss_list` to be formatted like:
```python
[
((0, 2, 4), HuberLoss()),
((1,3), QuaddraticLoss()
]
```

so that you can specify which loss applies to which columns.

With the change you've made you are now applying every loss function to every column which if you have mixed datatypes will be bad news.